persistence/test/ibc_test.go

```diff
@@ -120,3 +121,180 @@ func TestGetIBCStoreEntry(t *testing.T) {
 		})
 	}
 }
+
+type attribute struct {
+	key   []byte
+	value []byte
+}
+
+func TestIBCSetEvent(t *testing.T) {
+	db := NewTestPostgresContext(t, 1)
+	event := new(coreTypes.IBCEvent)
+	event.Topic = "test"
+	event.Height = 1
+	event.Attributes = append(event.Attributes, &coreTypes.Attribute{
+		Key:   []byte("testKey"),
+		Value: []byte("testValue"),
+	})
+	require.NoError(t, db.SetIBCEvent(event))
+
+	testCases := []struct {
+		name        string
+		height      uint64
+		topic       string
+		attributes  []attribute
+		expectedErr string
```

Comment:
```suggestion
		expectedErrStr *string
```

Make it a nil if there is no  error.